docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -0,0 +1,519 @@
+The workflow to make changes to your infrastructure generally involves updating your Terraform configuration file followed by a `terraform plan` and `terraform apply`.
+
+### Update the Terraform configuration
+Open the file `./example-1/modules/guestbook_service/main.tf` in a text editor and make the change to add a `scale` property with a value of `3` to the `layer0_service` section. For more information about the `scale` property, see [Layer0 Terraform Plugin](http://layer0.ims.io/reference/terraform-plugin/#service) documentation. The result should look like the below:
+
+example-1/modules/guestbook_service/main.tf
+
+```
+# Create a service named "guestbook"
+resource "layer0_service" "guestbook" {
+  name          = "guestbook"
+  environment   = "${layer0_environment.demo.id}"
+  deploy        = "${layer0_deploy.guestbook.id}"
+  load_balancer = "${layer0_load_balancer.guestbook.id}"
+  scale         = 3
+}
+
+```
+
+### Plan and Apply
+Execute the plan command to understand the changes that you will be making. Note that if you did not specify `scale`, it defaults to '1'.
+
+`terraform plan`
+
+Outputs:
+
+```
+...
+
+~ module.guestbook.layer0_service.guestbook
+    scale: "1" => "3"
+```
+
+Now run the following command to deploy your changes:
+
+`terraform apply`
+
+Outputs:
+
+```
+layer0_environment.demo: Refreshing state... (ID: demoenvbb9f6)
+data.template_file.guestbook: Refreshing state...
+layer0_deploy.guestbook: Refreshing state... (ID: guestbook.6)
+layer0_load_balancer.guestbook: Refreshing state... (ID: guestbo43ab0)
+layer0_service.guestbook: Refreshing state... (ID: guestboebca1)
+layer0_service.guestbook: Modifying... (ID: guestboebca1)
+  scale: "1" => "3"
+layer0_service.guestbook: Modifications complete (ID: guestboebca1)
+
+Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
+
+The state of your infrastructure has been saved to the path
+below. This state is required to modify and destroy your
+infrastructure, so keep it safe. To inspect the complete state
+use the `terraform show` command.
+
+State path: 
+
+Outputs:
+
+services = <guestbook_service_url>
+```
+
+To confirm your service has been updated to the desired scale, you can run the following layer0 command. Note desired scale for the guestbook service should be eventually be 3/3.
+
+`l0 service get guestbook1_guestbook_svc`
+
+Outputs:
+
+```
+SERVICE ID    SERVICE NAME  ENVIRONMENT  LOADBALANCER  DEPLOYMENTS  SCALE
+SERVICE ID    SERVICE NAME              ENVIRONMENT  LOADBALANCER             DEPLOYMENTS                  SCALE
+guestbo4fd3b  guestbook1_guestbook_svc  demo         guestbook1_guestbook_lb  guestbook1_guestbook_dpl:3*  1/3 (2)
+```
+
+As scale is parameter we are likely to change in the future, rather than hardcoding it to 3 as we have done just now, it would be better to use a variable to store  `service_scale`. The following Best Practices sections will show how you can achieve this.
+
+!!! Tip "Best Practices with Terraform + Layer0"
```

Comment:
@zpatrick it originally was, but it breaks the nav section such that all sections below it will no longer be shown in the nav section. The content directly under Best Practices wouldn't justify a h2 section either I thought. Which is how I ended up with the Note 🙁